application/CohortManager/src/Functions/Shared/Model/CohortDistributionParticipant.cs

```
@@ -100,10 +100,10 @@ public CohortDistributionParticipant(CohortDistribution cohortDistribution)
 
     public CohortDistribution ToCohortDistribution()
     {
-        return new CohortDistribution
+        var cohortDis = new CohortDistribution()
         {
             RequestId = GetRequestId(),
-            NHSNumber = long.Parse(NhsNumber),
+            NHSNumber = long.Parse(NhsNumber ?? "0"),
```

why?

---

yeah, but we want it to error, we don't want to suppress it and make it 0

---

I agree with will this doesn;t sit right we are suppressing a potential error here.